frontend/src/components/ButtonBuzzer.tsx

```diff
@@ -0,0 +1,58 @@
+import styled from "styled-components";
+
+type ButtonBuzzerProps = {
+    handleClick: () => void,
+    buttonText?: string,
+    color?: string
+    $position?: string;
+}
+
+export default function ButtonBuzzer(props: Readonly<ButtonBuzzerProps>) {
+    return (
+        <ButtonContainer $position={props.$position}>
+            <BuzzerButton type={"button"} onClick={props.handleClick} color={props.color}>
+                <StyledSpan color={props.color}>{props.buttonText}</StyledSpan>
+            </BuzzerButton>
+        </ButtonContainer>
+    )
+}
+
+const ButtonContainer = styled.div<{$position?: string}>`
+  display: flex;
+  margin: 40px 30px;
+  position: fixed;
+  bottom: 0;
+  right: ${({$position}) => $position === "right" ? 0 : undefined};
+  left: ${({$position}) => $position === "left" ? 0 : undefined};
```

Comment:
Would that be a better solution?

```suggestion
  ${({$position}) => `${$position}: 0`;
```

Since positions holds only `left` or `right` you can work with this Because you only need one of them. (How I understand this)

Reply:
changed it